### PR TITLE
🚑 Fix upload previews on mobile

### PR DESCRIPTION
**Pull request by GoOz**
Resize, crop & fit image previews while uploading them.

### CODE REVIEW

**Comment by enguerran** [comment]
[(3) [DRI] 📱 Améliorer le retour visuel lors de la sauvegarde de photo](https://trello.com/c/7uFiqx4D/526-3-dri-%F0%9F%93%B1-am%C3%A9liorer-le-retour-visuel-lors-de-la-sauvegarde-de-photo)

File: src/drive/mobile/styles/uploadprogression.styl [hunk 1]
```diff
@@ -33,9 +33,11 @@
         position absolute
         top 50%
         left 50%
-        max-width 3rem
-        max-height 3rem
         transform translateX(-50%) translateY(-50%)
+        width 3rem
+        height 3rem
+        object-fit cover
```
Comment (y-lohse) [inline src/drive/mobile/styles/uploadprogression.styl]: TIL. But are you sure about this change? It looks like `cover` might [crop the image](https://developer.mozilla.org/fr/docs/Web/CSS/object-fit)?

Comment (GoOz) [reply src/drive/mobile/styles/uploadprogression.styl]: That's the point. It's just a preview to know which image we're talking about but that's pretty much it and any image that is not a square will be cropped.

Comment (y-lohse) [reply src/drive/mobile/styles/uploadprogression.styl]: Oooh sorry, of course. Somehow in my mind this was a fix for the viewer in photo so I was a bit concerned. My bad!

**Pull request merged**